src/components/Checkbox/Checkbox.js

```diff
@@ -43,8 +50,8 @@ export default class Checkbox extends React.Component {
           }
         });
 
-    const required = <abbr class="chi-label__required" title="Required field">*</abbr>;
-    const optional = <abbr class="chi-label__optional" title="Optional field">(optional)</abbr>;
+    const required = <abbr className="chi-label__required" title="Required field">*</abbr>;
```

Comment:
These classes are repeatedly used in different components. Can you add them to the classes.js, please?